### DISCUSSION TITLE
Allow player controlled monsters to see their own inventory while keeping it hidden from others until death

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#6363] [Ideas and suggestions]
Most monsters have `inventoryaccessiblewhenalive="false"` , this means that their inventory cannot be accessed until they are killed, effectively making their inventory more of a "loot table", which works well and makes sense.

The problem with this is what if you were to control a monster that has this setting set to false, **you wouldn't be able to see your own inventory**. The only way around this would be to set `inventoryaccessiblewhenalive` to true, however this would make it so people can steal your items by just stunning and grabbing you, which may not be ideal.

It would be good if setting `inventoryaccessiblewhenalive` to false didn't prevent characters from accessing their own inventories.

**Comment by ghost** [comment]
maybe just create a whole new setting, it could be called inventoryvisiblewhenalive or something, though I'm not sure that's what you meant
